lib/std/ordsets.nim

```diff
@@ -0,0 +1,782 @@
+#
+#
+#            Nim's Runtime Library
+#        (c) Copyright 2012 Andreas Rumpf
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+## The ``ordsets`` module implements an efficient `Ordinal` set implemented as a
+## `sparse bit set`:idx:.
+##
+## Supported `A: Ordinal` types include enums and distincts with int base.
```

Comment:
true but incomplete (excludes char, bool etc).
instead, maybe just say: Supports any `Ordinal` type.

Reply:
So the change is to supports distincts with any ordinal base? I didn't check any other cases other than it but that makes sense. Feel free to add an edit, I'm away from my laptop till Friday evening

Reply:
I mean that `Ordinal` is already defined elsewhere so there's no need to specify again what it means (and excluding some members from its definition eg `char`, `bool` etc)